python/py-pycryptodome/Portfile

```diff
@@ -14,15 +14,15 @@ long_description    PyCryptodome is a self-contained Python package of \
                     low-level cryptographic primitives. \
                     PyCryptodome is a fork of PyCrypto.
 
-python.versions     27 35 36
+python.versions     27 35 36 37
 
 homepage            https://pypi.python.org/pypi/${python.rootname}/
 master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 distname            ${python.rootname}-${version}
```

Comment:
these are the default values `master_site` and `distname` set by the Python PG so please remove them from the Portfile

Reply:
same applies for `distname`

Reply:
You haven’t removed the distname line